File: URI Online Judge/1002_Area_do_Circulo.js
//Por Maxilliam Araujo
/*
PROBLEMA: 1002 - Área do Círculo
RESPOSTA: Accepted
LINGUAGEM: JavaScript (nodejs 8.4.0) [+2s] {beta}
TEMPO: 0.092s
TAMANHO: 216 Bytes
SUBMISSÃO: 11/07/2020 17:48:54
*/
//https://www.urionlinejudge.com.br/judge/pt/problems/view/1002

var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

const raio = parseFloat(lines.shift());
const area = 3.14159 * (raio * raio);

console.log('A='+ area.toFixed(4));

File: URI Online Judge/1002_Area_do_Circulo.c
//por Maximilliam Araujo
/*
PROBLEMA: 1002 - Área do Círculo
RESPOSTA: Accepted
LINGUAGEM: C (gcc 4.8.5, -O2 -lm) [+0s]
TEMPO: 0.000s
TAMANHO: 161 Bytes
SUBMISSÃO: 15/09/2015 23:58:26
*/
#include<stdlib.h>
#include<stdio.h>

int main(){
	double A,area;
	scanf("%lf",&A);
	area = (A*A)*3.14159;
	printf("A=%.4lf\n",area);	
	
	return 0;
}
